smartsim/_core/utils/telemetry/telemetry.py

```diff
@@ -310,6 +79,24 @@ def __init__(
         case_sensitive: bool = False,
         timeout_ms: int = 1000,
     ) -> None:
+        """Initialize the manifest event handler
+
+        :param exp_id: unique ID of the parent experiment executing a run
+        :type exp_id: str
+        :param pattern: a pattern that identifies the files whose
+        events are of interest by matching their name
+        :type pattern:  str
+        :param ignore_patterns: a pattern that identifies the files whose
```

Comment:
it could be either but my subclass can define it better. replaced w/
```
:param ignore_patterns: a pattern that identifies the files whose
events should be ignored
:type ignore_patterns:  Optional[List[str]]
```